src/navtypes.h

```diff
@@ -14,6 +14,12 @@ struct gpscoords_t {
 	double lon;
 };
 
+struct orientation_t {
+	double roll;
+	double pitch;
+	double yaw;
+};
```

Comment:
In some cases we represent orientation as a quaternion instead. Since RPY isn't the only way of representation orientation, maybe this should be called `eulerangles_t` or something?